### PR TITLE
Added topics courses page

### PR DESCRIPTION
**Pull request by marchdan**
Created a new header button to link to a page, Topics, which shows any course with a "Featured" tag

### CODE REVIEW

File: src/app/header/component.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
           <input #term id="searchbar" placeholder="Search Fall 2017" (keyup.enter)="search(term.value)"/>
         </span>
       </td>
+      <td class="sidelink" id="schedule-btn">
+        <a [routerLink]="['/courses']" [queryParams]="{ tags: ['Featured'] }">Topics<span class = "tiny">&nbsp; new!</span></a>
```
Comment (Bad-Science) [inline src/app/header/component.html]: `'featured'` should be lowercase

**Review by Bad-Science — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good, just one minor change requested.

File: src/app/header/component.html (hunk 1 above)
Comment (marchdan) [reply src/app/header/component.html]: Will correct when I can get to my computer

**Review by Bad-Science — APPROVED** [review]
Awesome, Thanks!

**Pull request merged**